conbench/app/_plots.py

```diff
@@ -513,7 +513,29 @@ def time_series_plot(history, benchmark, run, height=380, width=1100):
         x_axis_type="datetime",
         height=height,
         width=width,
-        tools=["pan", "zoom_in", "zoom_out", "reset", "tap"],
+        tools=[
+            # Allows for dragging the field of view, default drag action.
+            "pan",
+            # Allow for resetting the plot to original view.
+            "reset",
+            # This allows for benchmark-details-on-datapoint-click
+            "tap",
+            # Zoom in and out with mouse wheel, default wheel action
+            "wheel_zoom",
+            # Allow for drawing a box for zooming, not enabled by default,
+            # because the "pan" tool is enabled by default. Can be activated by
+            # clicking the icon on the toolbar (one can toggle manually between
+            # pan and box-zoom).
+            "box_zoom",
+        ],
+        # Bokeh toolbars can have at most one active tool from each kind of
+        # gesture (drag, scroll, tap).
+        # https://docs.bokeh.org/en/2.4.0/docs/user_guide/tools.html#setting-the-active-tools
+        active_drag="pan",
```

Comment:
Since the default view allows us to see all data on the plot, I think the default drag should be box_zoom. There's not really a point to panning left or up when you can see everything already.

Reply:
This is a great point. Made me think and compare a little more.

I think after all it's the combination of two tools that creates the navigation comfort zone or not. I found wheel_zoom+pan rather powerful, gives a clear and intuitive method to navigate the plot very quickly along all dimensions. Right after having zoomed in it's good to be able to pan to correct/adjust the field of view slightly or by a lot.

I have just tried wheel_zoom+box_zoom locally as default combo and I found that there's a double click that after box_zoom let's you quickly zoom back out again, so you can re-adjust the next box zoom. That's OK, too.

For now, my personal favorite is  wheel_zoom+pan. But I choose to not care too much yet and have pushed a commit to make this wheel_zoom+box_zoom :) That's fine!